packages/responsive/src/useMediaQueries.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react"
+import { useMediaQuery } from "react-responsive"
+import { useTheme } from "@kaizen/design-tokens"
+
+type Props = { [key: string]: string }
+
+export const useMediaQueries = (propQueries: Props) => {
+  const theme = useTheme()
+
+  // TODO: max-width usage needs to be -1px
+  const kaizenQueries = {
+    isSmall: useMediaQuery({
+      query: `(max-width: ${theme.layout.breakpoints.medium})`,
```

Comment:
We'd need a `- 1` here to make it consistent with the approach of "each breakpoint represents the minimum in its range, not the maximum of the range below it".

Reply:
Sorry, just noticed the bullet point addressing this in the todo stuff 🤦